package.json

```diff
@@ -1,15 +1,48 @@
 {
-  "name": "code-exercise",
-  "version": "1.0.0",
-  "description": "This is a shopping cart coding exercise",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "canned -p 5000 ./server"
-  },
-  "author": "",
-  "license": "MIT",
-  "dependencies": {
-    "canned": "^0.3.12"
-  }
-}
+    "name": "shopping-app",
+    "version": "1.0.0",
+    "description": "This is a shopping cart coding exercise",
+    "main": "index.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "compile:sass": "node-sass resources/sass/main.scss resources/css/style.css",
+        "start": "node ./bin/www",
+        "start:dev": "npm run compile:sass-dev"
+    },
+    "author": "Navin Tyagi",
+    "license": "MIT",
+    "devDependencies": {
+        "babel-preset-env": "^1.7.0",
+        "browser-sync-webpack-plugin": "^2.2.2",
+        "copy-webpack-plugin": "^5.0.2",
+        "css-loader": "^2.1.1",
+        "express": "^4.16.4",
+        "express-handlebars": "^3.0.2",
+        "extract-text-webpack-plugin": "^4.0.0-beta.0",
+        "file-loader": "^3.0.1",
+        "hbs": "^4.0.3",
+        "html-webpack-plugin": "^3.2.0",
+        "mini-css-extract-plugin": "^0.5.0",
+        "node-sass": "^4.11.0",
+        "postcss-loader": "^3.0.0",
+        "postcss-preset-env": "^6.6.0",
+        "sass-loader": "^7.1.0",
+        "style-loader": "^0.23.1",
+        "webpack": "^4.29.6",
+        "webpack-cli": "^3.3.0",
+        "webpack-dev-server": "^3.2.1"
+    },
+    "dependencies": {
+        "@babel/core": "^7.4.0",
+        "babel-loader": "^8.0.5",
+        "browser-sync": "^2.26.3",
+        "cookie-parser": "^1.4.4",
+        "fs": "0.0.1-security",
+        "handlebars-loader": "^1.7.1",
```

Comment:
Can you  re-organise your dev deps and deps

Reply:
done